.gitignore

```diff
@@ -0,0 +1,91 @@
+# Built style type definitions
+*.scss.d.ts
+*.graphql.d.ts
+
+#TS output
+packages/**/dist
+dist
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directory
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
+node_modules
+
+# Debug log from npm
+npm-debug.log
+**/npm-debug.log
+
+# Debug log from lerna
+lerna-debug.log
+
+# Debug logs from yarn
+**/yarn-error.log
+
+# IntelliJ IDE ignore
+.idea
+
+.DS_Store
+
+tmp
+**/test.log
+
+*.tsbuildinfo# Built style type definitions
+*.scss.d.ts
+*.graphql.d.ts
+
+#TS output
+packages/**/dist
+dist
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directory
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
+node_modules
+
+# Debug log from npm
+npm-debug.log
+**/npm-debug.log
+
+# Debug log from lerna
+lerna-debug.log
+
+# Debug logs from yarn
+**/yarn-error.log
+
+# IntelliJ IDE ignore
+.idea
+
+.DS_Store
+
+tmp
+**/test.log
+
+*.tsbuildinfo
```

Comment:
🔥

Reply:
Can we keep this one? I know it's not relevate at the moment but my plan is to migrate to typescript eventually.